admin/services/data-access/completed-check.data.service.js

```diff
@@ -1,10 +1,14 @@
 'use strict'
 
 const CompletedChecks = require('../../models/completed-checks')
+const sqlService = require('./sql.service')
+const TYPES = require('tedious').TYPES
+const R = require('ramda')
 const completedCheckDataService = {}
 
 /**
  * Query : find a completedCheck document that has not been marked
+ * @deprecated
```

Comment:
maybe just for consistency just add something like: @deprecated unused - no replacement